monitor/services/src/main/java/org/datacleaner/monitor/server/controllers/JobsFolderController.java

```diff
@@ -160,6 +161,29 @@
     }
 
     @RolesAllowed(SecurityRoles.JOB_EDITOR)
+    @RequestMapping(value = "/multiple_upload", method = RequestMethod.POST, produces = MediaType.TEXT_HTML_VALUE, consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
+    public String uploadMultipleAnalysisJobsToFolderHtml(@PathVariable("tenant") final String tenant,
```

Comment:
I created a new method with unique URL suffix "multiple_upload" not to change public API.